.github/workflows/issue-auto-assign.yml

```
@@ -0,0 +1,29 @@
+name: 'Auto assign maintainer to issue'
+on:
+  issues:
+    types: [opened]
+
+permissions:
+  issues: write
+
+jobs:
+  assign-user:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const assignees = ['mstoykov', 'imiric, 'codebien', 'olegbespalov', 'oleiade'];
```

Sorry, noticed this just now too.

```suggestion
            const assignees = ['mstoykov', 'imiric', 'codebien', 'olegbespalov', 'oleiade'];
```